R/importRDB1.r

```diff
@@ -93,10 +99,10 @@ importRDB1 <- function(obs_url, asDateTime=TRUE, convertType = TRUE, tz=""){
                           "America/Phoenix","America/Metlakatla","UTC"))
   }
   
-
   if(file.exists(obs_url)){
     doc <- obs_url
   } else {
+    # doc <- getWebServiceData(obs_url, add_headers(Accept-Encoding = "gzip"))
```

Comment:
I think the commented-out stuff is not in final release (but I do have to work on being better about not committing that!)

https://github.com/USGS-R/dataRetrieval/blob/master/R/importRDB1.r